src/broker/cas_execute.c

```diff
@@ -757,6 +761,7 @@ ux_prepare (char *sql_stmt, int flag, char auto_commit_mode, T_NET_BUF * net_buf
     }
   srv_handle->schema_type = -1;
   srv_handle->auto_commit_mode = auto_commit_mode;
+  cub_ddl_log_commit_mode (auto_commit_mode);
```

Comment:
Put an intentional blank line for readability.

```
srv_handle->auto_commit_mode = auto_commit_mode;

cub_ddl_log_commit_mode (auto_commit_mode);
```

Reply:
Thank you, i will change code as your recommend.